app/src/main/res/layout/patient_search_view.xml

```diff
@@ -7,6 +7,12 @@
   tools:context="org.simple.clinic.searchresultsview.PatientSearchView"
   tools:parentTag="RelativeLayout">
 
+  <ProgressBar
```

Comment:
The "Register as a new patient" button is still visible while the search results are loading. If the results take more than a few seconds to load (as it will on production because of the amount of data), it is possible that a user could tap on the button and register a patient before the search results load completely.

We should hide the button as well until the results load.